sdk/communication/AzureCommunication/Source/CommunicationCloudEnvironment.swift

```diff
@@ -28,24 +28,21 @@
 #endif
 import Foundation
 
-@objcMembers public class  CommunicationCloudEnvironment {
+@objcMembers public class  CommunicationCloudEnvironment: NSObject {
     private(set) var environmentValue: String
 
     public static let Public = CommunicationCloudEnvironment(environmentValue: "public")
     public static let Dod = CommunicationCloudEnvironment(environmentValue: "dod")
     public static let Gcch = CommunicationCloudEnvironment(environmentValue: "gcch")
 
-    init(environmentValue: String) {
+    public init(environmentValue: String) {
         self.environmentValue = environmentValue
     }
 
     static func fromModel(environmentModel: CommunicationCloudEnvironmentModel) -> CommunicationCloudEnvironment {
```

Comment:
@RezaJooyandeh Can you help explain a bit about who is going to call this method?

Reply:
This method is to be used by communication libraries like chat when they are trying to deserialize an instance of `CommunicationCloudEnvironmentModel` that has been in a response.